### DISCUSSION TITLE
SPA with Api?

### DISCUSSION THREAD

**Discussion by AnalogJ** [getlift/lift#274] [Constructs]
Is something like this possible with `lift`?
- SPA served on `/*` or `/web/*`
- Api (Lambda) calls on `/api/*`

Basically it would be very similar to the `server-side-website` construct type, but would enable some of the nice `single-page-app` redirects

![server-side-website(1)](https://user-images.githubusercontent.com/891875/197951201-c75fb531-68f2-43e8-a65b-60fe5e9a63a6.png)

**Comment by mnapoli** [comment]
Hi, this isn't possible at the moment. This has been brought up in #82
